docs/zkapps/advanced-snarkyjs/actions-and-reducer.mdx

```diff
@@ -11,7 +11,7 @@ Please note that zkApp programmability is not yet available on Mina Mainnet, but
 
 # Actions & Reducer
 
-Like events, **actions** are _public_ arbitrary information passed along with a zkApp transaction. However, actions give you an additional power: you can process previous actions in a smart contract! Under the hood, this is possible because we store a commitment to the history of dispatched actions on every account -- the **actionsHash**. It allows us to prove that the actions you process are, in fact, the actions that were dispatched to the same smart contract.
+Like events, **actions** are _public_ arbitrary information passed along with a zkApp transaction. However, actions give you an additional power: you can process previous actions in a smart contract! Under the hood, this is possible because we store a commitment to the history of dispatched actions on every account -- the **actionState**. It allows us to prove that the actions you process are, in fact, the actions that were dispatched to the same smart contract.
```

Comment:
```suggestion
Like events, **actions** are _public_ arbitrary information that are passed along with a zkApp transaction. However, actions give you additional power: you can process previous actions in a smart contract! Under the hood, this is possible because a commitment is stored to the history of dispatched actions on every account -- the **actionState**. It allows you to prove that the actions you process are, in fact, the actions that were dispatched to the same smart contract.
```